Cell.java

```diff
@@ -11,9 +13,20 @@ public class Cell extends JButton {
     public boolean isFlagged;
     public boolean isRevealed;
 
+    void markUnknown() {
+        setBackground(new ColorUIResource(255, 255, 0));
```

Comment:
Maybe change the way the buttons are colored?

Reply:
what colors should we use for each mark? I dont really know a lot about which colors work for colorblind

Reply:
i think the more important part is that the whole button should be covered in a bright / distinguishable shade because it is only the edges for now but something like RED(255, 0, 0), GREEN(0, 210, 10) and YELLOW(255, 255, 0), i.e slightly tweak green only and make the whole button the color